prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/jdbc/JdbcServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.parodos.tasks.jdbc;
+
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.jdbc.core.ColumnMapRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+
+class JdbcServiceImpl implements JdbcService {
+
+	@Override
+	public List<Map<String, Object>> query(String url, String statement) {
+		DriverManagerDataSource dataSource = new DriverManagerDataSource();
+		dataSource.setUrl(url);
+
+		JdbcTemplate jdbcTemplate = new JdbcTemplate(dataSource);
```

Comment:
this is repetitive. the 4 lines above repeat themselves in this class.

Reply:
extract lines 14-17 to a private function and reuse it in execute/update/query functions.

Reply:
fixed